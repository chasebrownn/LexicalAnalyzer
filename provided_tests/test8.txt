t9 ((i)|(x)).(q) #
" iq xq ix "