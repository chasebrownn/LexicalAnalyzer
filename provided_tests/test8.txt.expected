t9 , "iq"
t9 , "xq"
ERROR
